drivers/hud_twenty_twenty_to_twenty_twenty_two/app/models/hud_twenty_twenty_to_twenty_twenty_two/disability/transform.rb

```diff
@@ -0,0 +1,20 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudTwentyTwentyToTwentyTwentyTwo::Disability::Transform
+  module_function
+
+  def up(source_class, source_config, dest_class, dest_config)
+    Kiba.parse do
+      source source_class, source_config
+
+      transform(&:to_hash) # The CSV loader returns something that is not quite a hash
+      transform ::HudTwentyTwentyToTwentyTwentyTwo::Disability::AddAntiRetroviral
+
+      destination dest_class, dest_config
+    end
+  end
```

Comment:
These might also benefit from a base module, it seems the only thing that differs is which transformations need to be called, which I _think_ could be fairly easily moved to a method call.